.github/labeler.yml

```diff
@@ -44,3 +53,14 @@
 "CI-tfm-test":
   - "modules/tfm/**/*"
   - "samples/tfm/**/*"
+  - "tests/subsys/spm/secure_services/**/*"
```

Comment:
SPM secure services are unrelated to TFM
```suggestion
```

Reply:
It's not related to TF-M, but this test is still executed on the TF-M CI as there weren't really any other meaningful place to execute it. So even though it looks out of place I think we should keep it here, at least until we have a clearer scope for were the SPM tests will be executed.

Reply:
Aah ok, then of course we keep it.